pkg/util/azureclient/environments.go

```diff
@@ -58,6 +62,8 @@ var (
 		AppLensScope:             "https://microsoft.onmicrosoft.com/runtimehost",
 		AppLensTenantID:          "cab8a31a-1906-4287-a0d8-4eef66b95f6e",
 		Cloud:                    cloud.AzureGovernment,
+		PkiIssuerUrlTemplate:     "https://issuer.pki.azure.com/dsms/issuercertificates?getissuersv3&caName=%s",
+		PkiCaName:                "ame",
```

Comment:
its' `issuer.pki.azure.us` for fairfax. Besides, the ca name is ssladmin, I think.

Reply:
AppLens .us uses DigiCerts that are included in the docker container by default, and we dont have to fetch them.  I have updated the code to pass ""